tests/setup/challenges-data.test.js

```diff
@@ -15,8 +14,8 @@ var challenge = [{
 var query = challengesData(challenge);
 
 var expected = 'INSERT INTO challenges '
```

Comment:
You can call your endpoint which add a challenge - determine which id the challenge has been inserted to, then call  the getById endpoint or function to check the value of the data. I post this comment on the wrong part of the code sorry, this test might be better added to the one which add a challenge with the server.inject on "/"